.github/workflows/backend-api-spec-update.yml

```diff
@@ -1,18 +1,23 @@
 name: Update backend api spec generated by Kotlin Gradle Plugin
 
 on:
-  pull_request:
-    paths:
-      - 'save-backend/**/*Controller*'
+  schedule:
+    # triggers the workflow every day at 5:30 UTC
+    - cron: '30 5 * * *'
 
 jobs:
   update_backend_api_spec:
     runs-on: ubuntu-latest
     steps:
+      - uses: peterjgrainger/action-create-branch@v2.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: 'infra/update-backend-api-spec'
```

Comment:
what's about:
```
branch
Optional The name of the branch to create. Default "release-candidate". If your branch conains forward slashes (/) use the full branch reference. Instead of /long/branch/name use refs/heads/long/branch/name. It's an issue with the GitHub API https://gist.github.com/jasonrudolph/10727108
```?

Reply:
I saw it, but it's successfully created without /refs/heads

Reply:
However. I can add it, nevermind